clang/lib/Sema/SemaOpenMP.cpp

```diff
@@ -11,6 +11,7 @@
 ///
 //===----------------------------------------------------------------------===//
 
+#include "clang/Sema/SemaOpenMP.h"
```

Comment:
Sort it properly?

Reply:
Main module header goes first per our coding standard: https://llvm.org/docs/CodingStandards.html#include-style